packages/create-remix/templates/fly-stack/prisma/schema.prisma

```diff
@@ -8,16 +8,30 @@ generator client {
 }
 
 model User {
-  id       String @id @default(cuid())
-  email    String @unique
-  password String
+  id    String @id @default(cuid())
+  email String @unique
+
+  createdAt DateTime @default(now())
+  updatedAt DateTime @updatedAt
+
+  password Password?
   notes    Note[]
 }
 
+model Password {
+  user     User   @relation(fields: [userId], references: [id])
+  userId   String @unique
+  password String
```

Comment:
Calling this `hash` instead of `password` allows the code elsewhere to read: `user.password.hash` instead of `user.password.password` which I think is a bit nicer :)

```suggestion
  hash         String
```